x/wasm/keeper/querier_test.go

```diff
@@ -284,6 +284,127 @@ func TestQueryRawContractState(t *testing.T) {
 	}
 }
 
+func TestQueryContractsByCode(t *testing.T) {
+	ctx, keepers := CreateTestInput(t, false, AvailableCapabilities)
+	keeper := keepers.WasmKeeper
+	q := Querier(keeper)
+
+	// store contract
+	contract := StoreHackatomExampleContract(t, ctx, keepers)
+
+	createInitMsg := func(ctx sdk.Context, keepers TestKeepers) sdk.AccAddress {
+		_, _, verifierAddr := keyPubAddr()
+		fundAccounts(t, ctx, keepers.AccountKeeper, keepers.BankKeeper, verifierAddr, contract.InitialAmount)
+
+		_, _, beneficiaryAddr := keyPubAddr()
+		initMsgBz := HackatomExampleInitMsg{
+			Verifier:    verifierAddr,
+			Beneficiary: beneficiaryAddr,
+		}.GetBytes(t)
+		initialAmount := sdk.NewCoins(sdk.NewInt64Coin("denom", 100))
+
+		adminAddr := contract.CreatorAddr
+		label := "demo contract to query"
+		contractAddr, _, err := keepers.ContractKeeper.Instantiate(ctx, contract.CodeID, contract.CreatorAddr, adminAddr, initMsgBz, label, initialAmount)
+		fmt.Println(contract.CodeID)
+		require.NoError(t, err)
+		return contractAddr
+	}
+
+	// related link: https://github.com/CosmWasm/wasmd/issues/1559
+	// When not set with a block, the contract will return results based on the order of address rather than instantiation (timestamp).
```

Comment:
> based on the order of address

What does this mean?

Reply:
It means that the order of contract address

Reply:
Understood, thank you!